app/gui/src/dashboard/layouts/AssetsTable.tsx

```diff
@@ -449,7 +453,7 @@ export default function AssetsTable(props: AssetsTableProps) {
   const createSecretMutation = useMutation(backendMutationOptions(backend, 'createSecret'))
   const updateSecretMutation = useMutation(backendMutationOptions(backend, 'updateSecret'))
   const createDatalinkMutation = useMutation(backendMutationOptions(backend, 'createDatalink'))
-  const uploadFileMutation = useMutation(backendMutationOptions(backend, 'uploadFile'))
+  const uploadFileMutation = useUploadFileMutation(backend)
```

Comment:
if it's a `mutation`, then I think it should, at least, expose `isMutating`, `isError`, `error` fields